oidc-api/build.gradle

```diff
@@ -9,6 +9,7 @@ version "unspecified"
 dependencies {
 
     implementation project(path: ':ipv-api')
+    implementation project(path: ':orchestration-shared')
```

Comment:
I wonder if the ipv `implementation` is only here because it has an `implementation` and `testImplementation` - seems the more common pattern is to define it in the list below (and feels a bit tidier)